include/openthread/link.h

```diff
@@ -306,226 +306,278 @@ OTAPI void OTCALL otLinkSetPollPeriod(otInstance *aInstance, uint32_t aPollPerio
+ * @param[in]  aRss      The Rss value (in dBm) to set.
  *
- * @returns TRUE if the MAC whitelist is enabled, FALSE otherwise.
+ * @sa otLinkGetRssInFilter
+ * @sa otLinkAddRssInFilterEntry
+ * @sa otLinkRemoveRssInFilterEntry
+ * @sa otLinkClearRssInFilter
+ * @sa otLinkGetNextRssInFilterEntry
+ * @sa otLinkAddFilterEntry
  *
- * @sa otLinkAddWhitelist
- * @sa otLinkAddWhitelistRssi
- * @sa otLinkRemoveWhitelist
- * @sa otLinkClearWhitelist
- * @sa otLinkGetWhitelistEntry
- * @sa otLinkSetWhitelistEnabled
  */
-OTAPI bool OTCALL otLinkIsWhitelistEnabled(otInstance *aInstance);
+void otLinkSetRssInFilter(otInstance *aInstance, int8_t aRss);
 
 /**
- * Add an IEEE 802.15.4 Extended Address to the MAC blacklist.
+ * This method gets the default RssIn for all received messages.
  *
  * @param[in]  aInstance A pointer to an OpenThread instance.
- * @param[in]  aExtAddr  A pointer to the IEEE 802.15.4 Extended Address.
  *
- * @retval OT_ERROR_NONE     Successfully added to the MAC blacklist.
- * @retval OT_ERROR_NO_BUFS  No buffers available for a new MAC blacklist entry.
+ * @returns The default RssIn (in dBm) if configured or OT_RSS_OVERRIDE_DISABLED if not.
+ *
+ * @sa otLinkSetRssInFilter
+ * @sa otLinkAddRssInFilterEntry
+ * @sa otLinkRemoveRssInFilterEntry
+ * @sa otLinkClearRssInFilter
+ * @sa otLinkGetNextRssInFilterEntry
+ * @sa otLinkAddFilterEntry
  *
- * @sa otLinkRemoveBlacklist
- * @sa otLinkClearBlacklist
- * @sa otLinkGetBlacklistEntry
- * @sa otLinkIsBlacklistEnabled
- * @sa otLinkSetBlacklistEnabled
  */
-OTAPI otError OTCALL otLinkAddBlacklist(otInstance *aInstance, const uint8_t *aExtAddr);
+int8_t otLinkGetRssInFilter(otInstance *aInstance);
 
 /**
- * Remove an IEEE 802.15.4 Extended Address from the MAC blacklist.
+ * This method adds an Extended Address to the RssInFilter and sets the RssIn (in dBm) for all received messages
+ * from the Extended Address.
  *
- * @param[in]  aInstance A pointer to an OpenThread instance.
- * @param[in]  aExtAddr  A pointer to the IEEE 802.15.4 Extended Address.
+ * @param[in]  aInstance    A pointer to an OpenThread instance.
+ * @param[in]  aExtAddress  A pointer to the IEEE 802.15.4 Extended Address.
+ * @param[in]  aRss         The RssiIn value (in dBm) to set.
+ *
+ * @retval OT_ERROR_NONE     Successfully added @p aExtAddress to the RssInFilter with @p aRss.
+ * @retval OT_ERROR_NO_BUFS  No availabl entry.
+ *
+ * @sa otLinkSetRssInFilter
+ * @sa otLinkGetRssInFilter
+ * @sa otLinkRemoveRssInFilterEntry
+ * @sa otLinkClearRssInFilter
+ * @sa otLinkGetNextRssInFilterEntry
+ * @sa otLinkAddFilterEntry
  *
- * @sa otLinkAddBlacklist
- * @sa otLinkClearBlacklist
- * @sa otLinkGetBlacklistEntry
- * @sa otLinkIsBlacklistEnabled
- * @sa otLinkSetBlacklistEnabled
  */
-OTAPI void OTCALL otLinkRemoveBlacklist(otInstance *aInstance, const uint8_t *aExtAddr);
+otError otLinkAddRssInFilterEntry(otInstance *aInstance, const otExtAddress *aExtAddress, int8_t aRss);
```

Comment:
How about passing `NULL` for `aExtAddress` to set the default received signal strength?  Then we can have a common set of APIs for modifying specific entries and the default.